cmd/gnoland/main.go

```diff
@@ -42,19 +35,81 @@ var flags struct {
 	rootDir               string
 }
 
-func runMain(args []string) error {
-	fs := flag.NewFlagSet("gnoland", flag.ExitOnError)
-	fs.BoolVar(&flags.skipFailingGenesisTxs, "skip-failing-genesis-txs", false, "don't panic when replaying invalid genesis txs")
-	fs.BoolVar(&flags.skipStart, "skip-start", false, "quit after initialization, don't start the node")
-	fs.StringVar(&flags.genesisBalancesFile, "genesis-balances-file", "./gnoland/genesis/genesis_balances.txt", "initial distribution file")
-	fs.StringVar(&flags.genesisTxsFile, "genesis-txs-file", "./gnoland/genesis/genesis_txs.txt", "initial txs to replay")
-	fs.StringVar(&flags.chainID, "chainid", "dev", "chainid")
-	fs.StringVar(&flags.rootDir, "root-dir", "testdir", "directory for config and data")
-	fs.StringVar(&flags.genesisRemote, "genesis-remote", "localhost:26657", "replacement for '%%REMOTE%%' in genesis")
-	fs.Parse(args)
+func main() {
+	cfg := &gnolandCfg{}
+
+	cmd := commands.NewCommand(
+		"gnoland",
+		commands.Metadata{
+			Name:       "",
+			ShortUsage: "[flags] [<arg>...]",
```

Comment:
```suggestion
			ShortUsage: "[flags] [<arg>...]",
```

skip this?

Reply:
You're suggesting we drop the usage format for the `gnoland` command?

The help output would look like this:

```bash
USAGE


Starts the gnoland blockchain node

FLAGS
  -chainid dev                                                   the ID of the chain
  -genesis-balances-file ./gnoland/genesis/genesis_balances.txt  initial distribution file
  -genesis-remote localhost:26657                                replacement for '%%REMOTE%%' in genesis
  -genesis-txs-file ./gnoland/genesis/genesis_txs.txt            initial txs to replay
  -root-dir testdir                                              directory for config and data
  -skip-failing-genesis-txs=false                                don't panic when replaying invalid genesis txs
  -skip-start=false                                              quit after initialization, don't start the node


```

Reply:
Ok, no, it looks good.